DEPLOYMENT_INDEX.md

```
@@ -72,29 +74,23 @@
 
 ### Quick Start Tools
 
-7. **[deploy.sh](./deploy.sh)** - Interactive deployment helper
+7. **[CLOUDFLARE_DEPLOY.md](./CLOUDFLARE_DEPLOY.md)** - Cloudflare DNS + CDN
```

markdownlint-cli2 (0.20.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 7; Style: 1/2/3

(MD029, ol-prefix)